velox/vector/fuzzer/tests/VectorFuzzerTest.cpp

```diff
@@ -226,6 +227,18 @@ TEST_F(VectorFuzzerTest, constants) {
   ASSERT_EQ(VectorEncoding::Simple::CONSTANT, vector->encoding());
   ASSERT_FALSE(vector->mayHaveNulls());
 
+  auto shortDecimalType = fuzzer.randShortDecimalType();
```

Comment:
How do you plan to use these new methods? Don't we need this logic integrated into randType()?

Reply:
It needs to be and I have opened an issue for the same. Also, you suggested to isolate the changes to randType in a separate. I have opened an issue for this: https://github.com/facebookincubator/velox/issues/3942.

This change only adds capability to generate random DECIMALS and random unscaled values.
The random unscaled values are getting use in this PR:https://github.com/facebookincubator/velox/pull/3755/files now.